packages/shared/src/request/Interceptor.ts

```
@@ -95,3 +96,4 @@
       platformEnv.buildNumber as string,
   };
 }
+ 
```

ESLint

[error] Delete `·`

(prettier/prettier)